.github/workflows/ci-internal-tests.yml

```diff
@@ -33,6 +33,16 @@ jobs:
                 key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
                 restore-keys: |
                   ${{ runner.os }}-gradle-
+            - name: Cache Node.js modules
+              uses: actions/cache@v2
+              with:
+                path: |
+                  ~/.npm
+                  **/node_modules
```

Comment:
what about tools under node projects? is there any point to cache them too?

Reply:
We can add in the future, The node distro is small (9 mb).